src/status_im/ui/screens/intro/views.cljs

```diff
@@ -217,108 +244,357 @@
       (i18n/label :t/processing)]]
     [password-container confirm-failure? view-width]))
 
-(defn enable-fingerprint []
-  [vector-icons/icon :main-icons/fingerprint
-   {:container-style {:align-items :center
-                      :justify-content :center}
-    :width 76 :height 84}])
-
 (defn enable-notifications []
   [vector-icons/icon :main-icons/bell {:container-style {:align-items :center
                                                          :justify-content :center}
                                        :width 66 :height 64}])
 
-(defn bottom-bar [{:keys [step generating-keys? weak-password? encrypt-with-password?
+(defn bottom-bar [{:keys [step weak-password? encrypt-with-password?
```

Comment:
it was already there so let's not change it but I think that in the future we should avoid having these big step based components